SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -892,6 +892,10 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 void FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentButtonClicked()
 {
 	VerifyAndStartDeployment();
+	// UNR-4334 - Also plug into the new runtime fetching using SpatialPackageManager
+	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+	const FString RuntimeVersion = SpatialGDKEditorSettings->GetSelectedRuntimeVariantVersion().GetVersionForLocal();
+	FSpatialPackageManager::TryFetchRuntimeBinary(RuntimeVersion);
```

Comment:
Here's a question. What happens if the runtime and inspector binaries don't exist (so not yet downloaded) and you try and start a local deployment and inspector at the same time? Is the current implementation ok with this or will the calls interrupt each other? Give it a test please.

Reply:
Tested & all good!